router/router.go

```diff
@@ -184,7 +203,33 @@ func (rt *HandleT) workerProcess(worker *workerT) {
 
 			deliveryTimeStat.Start()
 			if misc.ContainsString(customDestinations, job.CustomVal) {
-				respStatusCode, respStatus, respBody = customdestinationmanager.Send(job.EventPayload, job.CustomVal)
+				key := paramaters.SourceID + "-" + paramaters.DestinationID
```

Comment:
We should move this code out of router. May be into custom destination manager.

Reply:
Incorporated.